mk/js.mk

```diff
@@ -1,20 +1,27 @@
 default: help
 
 PATH := node_modules/.bin:$(PATH)
+NODE_BINDIR ?= ./node_modules/.bin/
 SHELL ?= /bin/bash -o pipefail
+SYSTEMATIC_PATH ?= node_modules/systematic
+
+CONF_INI = systematic.ini
+READINI = $(NODE_BINDIR)readini
 
 export NODE_PATH := $(shell pwd):$(NODE_PATH)
 
 # Customizable variables
 #
-IS_APP ?= no
+IS_APP ?= $(shell $(READINI) $(CONF_INI) build.is_app yes)
+SYSTEMATIC_PROFILE ?= $(shell $(READINI) $(CONF_INI) build.profile vanillajs)
+PACKAGE ?= $(shell $(READINI) $(CONF_INI) package.name "No name, configure systematic.ini")
```

Comment:
voiiiilà ;) mais ça ne plante pas, si ?  A priori sans valeur par défaut c'est à readini de retourner un code d'erreur générique du genre "Error: no default value defined for 'package.name': please define it in your configuration"